bundles/org.openhab.binding.kaleidescape/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,655 @@
+
+	<!-- Kaleidescape Movie Player Thing -->
+	<thing-type id="player">
+		<label>Kaleidescape</label>
+		<description>
+			A Movie Player Zone in a Kaleidescape System
+		</description>
+
+		<channel-groups>
+			<channel-group id="ui" typeId="ui">
+				<label>User Interface</label>
+				<description>Controls and Information for the Player's On Screen Interface</description>
+			</channel-group>
+			<channel-group id="music" typeId="music">
+				<label>Music Zone</label>
+				<description>Controls and Information for the Player's Music Zone</description>
+			</channel-group>
+			<channel-group id="detail" typeId="detail">
+				<label>Content Details</label>
+				<description>Contains Metadata About a Selected Item</description>
+			</channel-group>
+		</channel-groups>
+
+		<properties>
+			<property name="Component Type">unknown</property>
+			<property name="Friendly Name">unknown</property>
+			<property name="Serial Number">unknown</property>
+			<property name="Control Protocol ID">unknown</property>
+			<property name="System Version">unknown</property>
+			<property name="Protocol Version">unknown</property>
+		</properties>
+
+		<config-description>
+			<parameter name="componentType" type="text" required="true">
+				<label>Component Type</label>
+				<description>Choose Type of Kaleidescape Component</description>
+				<limitToOptions>true</limitToOptions>
+				<options>
+					<option value="Player">Premiere (Any KPlayer, M Class [M300, M500, M700]) or Cinema One 1st Gen</option>
+					<option value="Cinema One">Cinema One 2nd Gen</option>
+					<option value="Alto">Alto</option>
+					<option value="Strato">Strato</option>
+				</options>
+				<default>Player</default>
+			</parameter>
+			<parameter name="host" type="text" required="false">
+				<context>network-address</context>
+				<label>Address</label>
+				<description>Host Name or IP Address of the Kaleidescape Component.</description>
+			</parameter>
+			<parameter name="port" type="integer" min="1" max="65535" required="false">
+				<label>Port</label>
+				<description>Communication Port for IP Connection to the Kaleidescape Component.</description>
+				<default>10000</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="serialPort" type="text" required="false">
+				<context>serial-port</context>
+				<label>Serial Port</label>
+				<description>(Optional) Serial Port to Use for Connecting Directly to an Individual Kaleidescape Component.</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="updatePeriod" type="integer" min="0" max="1" unit="s" required="false">
+				<label>Update Period</label>
+				<description>Tells the Component How Often Time Status Updates Should Be Sent; Values Greater Than 1 Are Not Yet
+					Implmented by the Protocol. Setting to 1 May Impact openHAB System Performance Due
+					to Constant Updates While Content
+					Playing.</description>
```

Comment:
Only labels should be capitalized. Same for below.

Reply:
Thanks. It did look awkward.